apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1765,10 +1769,6 @@ export const platform: NavMenuConstant = {
       url: undefined,
       items: [
         { name: 'Regions', url: '/guides/platform/regions' },
-        {
```

Comment:
Could you addd a redirect for this? It goes in this file: https://github.com/supabase/supabase/blob/master/apps/www/lib/redirects.js